### DISCUSSION TITLE
showcase, look what I did

### DISCUSSION THREAD

**Discussion by SuddenDevelopment** [mmmrqs/bl_ui_widgets#5] [General]
![image](https://user-images.githubusercontent.com/138704/210255474-ea53db20-c945-48c6-b565-c540aad01eed.png)

**Comment by SuddenDevelopment** [comment]
maybe more significant than how it looks is it's all configured dynamically from json for button groups and buttons

**Reply by mmmrqs** [reply]
Wow.  I am very well impressed by the interface you've made.  It is beautiful.  Makes me happy to see my tool being useful to others.  Please check the response I've just posted in Git's issue/discussion section.  I tried the demo again in my Blender 3.3 installation and the button's tooltips worked fine.  There is a checkbox under Blender's Preferences -> Interface panel that controls whether User Tooltips should be displayed or not, and my library looks at that setup in order to follow suit.  Yours may be unchecked.

Thanks for sharing it.  Much appreciated.
